Test/allocated1/dafny0/BitvectorsMore.dfy

```diff
@@ -1,3 +1,3 @@
-// RUN: %exits-with 4 %dafny /verifyAllModules /allocated:1 /print:"%t.print" /env:0 "%s" > "%t"
+// RUN: %exits-with 4 %dafny /errorLimit:10 /verifyAllModules /allocated:1 /print:"%t.print" /env:0 "%s" > "%t"
```

Comment:
Just to be clear, this _increases_ the error limit beyond the default of 5, so that we collect all errors that happen. I just looked, though, and using `0` would tell it to find as many as possible, so that'd be a better choice here, and everywhere in the test suite.

I generally agree that tests with a large number of errors should be broken up, but the particular case where this was necessary feels, to me, better as a unit than multiple pieces.

Reply:
Yes - we should make errorLimit=0 the default in the tests for everything -- both in lit and in the CI tests. I'll add that to my Dafny 4 run and see if we catch anything that way.